src/main/java/io/github/minecraftcursedlegacy/api/networking/PluginChannel.java

```diff
@@ -0,0 +1,41 @@
+package io.github.minecraftcursedlegacy.api.networking;
+
+import io.github.minecraftcursedlegacy.impl.networking.PluginMessagePacket;
+import net.minecraft.client.Minecraft;
+import net.minecraft.entity.player.ServerPlayer;
+import net.minecraft.network.PacketHandler;
+
+public abstract class PluginChannel {
+	/**
+	 * Returns the unique channel identifier of this channel in the form of modid:channelname
+	 * This should always have the same return value.
+	 * @return Channel Identifier
+	 */
+	public abstract String getChannelIdentifier();
+	
+	/**
+	 * Called when this channel recieves a packet
```

Comment:
full stop at the end of javadoc sentences

Reply:
no that's lame